src/test/app/AMM_test.cpp

```diff
@@ -6359,6 +6598,7 @@ struct AMM_test : public jtx::AMMTest
             STAmount const goodUsdBIT;
             STAmount const goodUsdBITr;
             IOUAmount const lpTokenBalance;
+            std::optional<IOUAmount> const lpTokenBalanceAlt;
```

Comment:
fix compilation error in gcc 14 or higer

```suggestion
            std::optional<IOUAmount> const lpTokenBalanceAlt = {};
```